cfme/fixtures/soft_assert.py

```diff
@@ -103,13 +112,13 @@ def handle_assert_artifacts(request, fail_message=None):
         return
     if not fail_message:
         short_tb = '{}'.format(sys.exc_info()[1])
-        short_tb = short_tb.encode('base64')
```

Comment:
Q: could you please explain the reason for using base64 module instead of .encode ?

Reply:
its a binary encoding that cant be applied on text in python3